Mathlib/Data/Int/GCD.lean

```diff
@@ -169,9 +169,11 @@ namespace Int
 
 theorem gcd_def (i j : ℤ) : gcd i j = Nat.gcd i.natAbs j.natAbs := rfl
 
-protected theorem coe_nat_gcd (m n : ℕ) : Int.gcd ↑m ↑n = Nat.gcd m n :=
+protected theorem natCast_gcd (m n : ℕ) : Nat.gcd m n = Int.gcd ↑m ↑n :=
   rfl
-#align int.coe_nat_gcd Int.coe_nat_gcd
+#align int.coe_nat_gcd Int.natCast_gcd
+
+@[deprecated (since := "2024-05-25")] alias coe_nat_gcd := Int.natCast_gcd
```

Comment:
Please keep the original direction for the deprecated version

Reply:
I've split that change to #13221

Reply:
Oh, I see what you mean. I had assumed `Int.gcd` what `Int`-valued